src/refdb_fs.c

```diff
@@ -63,6 +66,18 @@ typedef struct refdb_fs_backend {
 	uint32_t direach_flags;
 } refdb_fs_backend;
 
+/*
+ * List of symbolic reference names that are always stored in the
+ * gitdir. This is required such that we are able to correctly
+ * resolve these references via the gitdir instead of the
+ * commondir when working trees are used.
+ */
+static const char *symrefs[] = {
+    "HEAD",
+    "ORIG_HEAD",
+    NULL,
+};
```

Comment:
Well, those aren't really *refs*, per se, they're sort of ref-like but in a non-ref irregular format and should be outside the scope of the refdb.

Though to be completely honest, I would have expected the same was true for `ORIG_HEAD`.  Do we really use the refdb infrastructure for `ORIG_HEAD`?